src/ext/conversion/importProperties.ts

```diff
@@ -0,0 +1,28 @@
+import conversion from 'qlik-object-conversion';
+
+import data from '../../qae/data';
+import { ExportFormat, PropTree } from '../../types';
+
+const importProperties = (
+  exportFormat: ExportFormat,
+  initialProperties: EngineAPI.IGenericHyperCubeProperties,
+  extension: any,
```

Comment:
Was it not possible to use `unknown` instead?

Reply:
as there is 
```
defaultPropertyValues: {
      defaultDimension: extension.getDefaultDimensionProperties(),
      defaultMeasure: extension.getDefaultMeasureProperties(),
    },
```